### PR TITLE
update scripts

### PR DESCRIPTION
**Pull request by john0879**
Updated the scripts to report coverage correctly.

### CODE REVIEW

File: scripts/test.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 SOLUTION=$1
 
-dotnet test $SOLUTION --logger:trx --results-directory:/src/TestResults /p:CollectCoverage=true /p:CoverletOutputFormat=json%2Clcov%2Copencover /p:CoverletOutput=/src/TestResults/lcov
\ No newline at end of file
+dotnet test $SOLUTION --logger:trx --results-directory:/src/TestResults -m:1 /p:CollectCoverage=true /p:CoverletOutputFormat=json%2Clcov%2Copencover /p:CoverletOutput=/src/TestResults/ /p:MergeWith=/src/TestResults/coverage.json
```
Comment (mfickanderson) [inline scripts/test.sh]: documentation has the `-m:1` at the end

Comment (mfickanderson) [reply scripts/test.sh]: i also read that -p: is better w/ bash ```dotnet test $SOLUTION --logger:trx --results-directory:/src/TestResults -p:CollectCoverage=true -p:CoverletOutput=/src/TestResults/ -p:MergeWith=/src/TestResults/coverage.json -p:CoverletOutputFormat=opencover%2Cjson -m:1```

Comment (john0879) [reply scripts/test.sh]: OK. Last I saw there was a bug that required `-m:1` to come immediately after `test`. Didn't think it mattered anymore. I don't really care where it is so I'll move it if you saw reason to. `/p` came from the coverlet examples. I'll change those.

**Review by mfickanderson — APPROVED** [review]
LGTM

**Pull request merged**